faq-site-client/src/views/account/CreateAccount.vue

```diff
@@ -1,60 +1,64 @@
-                        name="password"
-                        :append-icon="userData.passwordvisible ? 'mdi-eye' : 'mdi-eye-off'"
-                        @click:append="() => (userData.passwordvisible = !userData.passwordvisible)"
-                        :type="userData.passwordvisible ? 'password' : 'text'"
-                        v-validate="'required|min:8'"
-                        box
-                        required
-                        @change="userData.passworderror = ''"
-                        @keyup.enter="doLogin"
-                ></v-text-field>
-                <v-text-field
-                        label="First name"
-                        v-model="userData.fistname"
-                        :error-messages="errors.collect('firstname')"
-                        name="firstname"
-                        v-validate="'required'"
-                        required
-                        box
-                ></v-text-field>
-                <v-text-field
-                        label="Last name"
-                        v-model="userData.lastname"
-                        :error-messages="errors.collect('lastname')"
-                        name="lastname"
-                        v-validate="'required'"
-                        required
-                        box
-                ></v-text-field>
-                <div>
-                    <v-btn depressed color="primary" @click="doCreateAccount">Create account</v-btn>
-                </div>
-            </v-card-text>
-        </v-card>
-    </v-flex>
+    <v-container fluid fill-height class="grey lighten-4">
+        <v-layout justify-center align-center>
+            <v-flex shrink>
+                <v-card>
+                    <v-card-title class="title font-weight-regular justify-space-between">
+                        <span>Create account</span>
+                    </v-card-title>
+                    <v-card-text>
+                        <v-text-field
+                                label="Email"
+                                v-model="userData.email"
+                                :error-messages="errors.collect('email') + userData.emailerror"
+                                name="email"
+                                v-validate="'required|checkTUEmail'"
+                                suffix="@student.tudelft.nl"
+                                required
+                                box
+                                @change="userData.emailerror = ''"
+                                @keyup.enter="doLogin"
+                        ></v-text-field>
+                        <v-text-field
+                                label="Password"
+                                v-model="userData.password"
+                                :error-messages="errors.collect('password')"
+                                name="password"
+                                :append-icon="userData.passwordvisible ? 'mdi-eye' : 'mdi-eye-off'"
```

Comment:
These ternary statements are the wrong way round in here and in login

Reply:
Thanks 👍